jjb/jobs/qpc-nightly-automation.yaml

```diff
@@ -40,11 +46,11 @@
                 echo "INSECURE_REGISTRY=\"--insecure-registry ${DOCKER_REGISTRY}\"" >> docker.conf
                 sudo cp docker.conf /etc/sysconfig/docker
                 sudo systemctl start docker
-                sudo docker pull "${DOCKER_REGISTRY}/quipucords/quipucords:1.0.0"
+                sudo docker load -i ${WORKSPACE}/quipucords*.tar.gz
                 # make log dir to save server logs
                 mkdir ${WORKSPACE}/log
                 # run container binding local ssh keys and log dir to the container
-                sudo docker run -d -p "443:443" -v /tmp:/tmp -v /home/jenkins/.ssh:/home/jenkins/.ssh -v ${WORKSPACE}/log:/var/log -i "${DOCKER_REGISTRY}/quipucords/quipucords:latest"
+                sudo docker run -d -p "443:443" -v /tmp:/tmp -v /home/jenkins/.ssh:/home/jenkins/.ssh -v ${WORKSPACE}/log:/var/log -i "quipucords:1.0.0"
```

Comment:
@chambridge `1.0.0` is the correct tag to use, right? Ot is there a different tag for whatever is the latest master version?

Reply:
`1.0.0` is correct.